lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -71,13 +73,19 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
   Widget build(BuildContext context) {
     return StoreConnector(
         builder: (BuildContext context, MySpaceViewModel viewModel) => Scaffold(
+              key: _scaffoldKey,
               appBar: AppBar(
+                automaticallyImplyLeading: false,
                 title: Text(
                     AppLocalizations.of(context).my_space_title,
                     style: TextStyle(fontSize: 24, color: Colors.white)),
                 centerTitle: true,
                 backgroundColor: AppColor.primaryColor,
+                leading: IconButton(
+                    icon: SvgPicture.asset(imagePath.icLinShareMenu),
+                    onPressed: () => _scaffoldKey.currentState.openDrawer()),
               ),
+              drawer: SideMenuDrawerWidget(),
```

Comment:
you are adding `SlideMenuDrawerWidget` to `MySpaceWidget`. What happen if I have another screen shows up like `SharedSpace` screen, this case we will need to implement drawer again, that's not good.
I suggest you implement drawer inside `HomeWidget` and handle transition there.

Reply:
cool

Reply:
Having a persisting drawer in Flutter seems really hard, putting it in HomeWidget means combining 2 widgets My Space and Custom Scaffold with Drawer and I don't know if it's possible.

My idea is to make a custom scaffold and extends it in needed widgets but It seems tough

Reply:
How about we have HomeWidget with a `AppBar` and now mySpace don't have the appear. Its only a child of HomeWidget (below app bar). this way we can have a unique drawer only

Reply:
Can you remind me what is HomeWidget used for ? Is it the first widget opened ?

Reply:
I think `AppBar` can only be inside `Scaffold`, we can't externalize it.

Reply:
> Can you remind me what is HomeWidget used for ? Is it the first widget opened ? 

You can imagine that its like a MainActivity in android, and then we have fragments inside it. I like the same way in flutter. In this case is HomeWidget

Reply:
> I think `AppBar` can only be inside `Scaffold`, we can't externalize it. 

Yes, I mean now only HomeWidget has the `AppBar`. After that you can remove `AppBar` from `MySpace`

Reply:
So you want to put the `Scaffold` in HomeWidget and put only the `Body` in My Space ?

Reply:
Yes, that's exactly what I mean 💯